core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -221,4 +225,18 @@ private[data] trait OneAndLowPriority2 extends OneAndLowPriority1 {
     }
 }
 
+private[data] trait OneAndLowPriority3 extends OneAndLowPriority2 {
+  implicit def catsDataTraverse1ForOneAnd[F[_]](implicit F: Traverse[F], F2: MonadCombine[F]): Traverse1[OneAnd[F, ?]] =
+    new NonEmptyReducible[OneAnd[F, ?], F] with Traverse1[OneAnd[F, ?]] {
+      def traverse1[G[_], A, B](fa: OneAnd[F, A])(f: (A) => G[B])(implicit G: Apply[G]): G[OneAnd[F, B]] = {
+        import cats.syntax.cartesian._
+
+        fa.map(a => Apply[G].map(f(a))(OneAnd(_, F2.empty[B])))(F)
+          .reduceLeft(((acc, a) => (acc |@| a).map((x: OneAnd[F, B], y: OneAnd[F, B]) => x.combine(y))))
+      }
+
```

Comment:
Can you add the implementation of `traverse` used in the `Traverse` instance above here as well?